### ISSUE TITLE
Curation and "Mappings"

### ISSUE THREAD

**Issue by David-Berson** [serre-lab/webknossos#2]
I can curate the segmentation data in wK, but I am still clueless about whether, as I do that, I am altering the archival "first draft" segmentation data (probably not the right thing to do) and how to manage the 'mappings' that are described in the documentation ( https://docs.webknossos.org/guides/volume_annotation#volume-annotations ) and which I've actually gotten to work on German wK before (can't remember how).  Presumably we want to build up a library of suggested curatorial changes, use that to develop our latest/greatest curated version, and also keep the original draft segmentation for reference (on wK).

**Comment by drewlinsley** [comment]
Are you able to edit the segmentation layer? I'm having trouble figuring out how to do this. I can see the segmentation in view mode, but it disappears in annotation mode.

I will investigate extracting the curations. I can't figure this out just yet either. @paulobaptista and I also need to figure out daily backups via database dumps.

**Comment by David-Berson** [comment]
Yeah I can edit when I create an annotation. But I have no idea where my
version lives. Mappings?
